sortinghat/cmd/unify.py

```diff
@@ -85,13 +92,13 @@ def run(self, *args):
 
         code = self.unify(params.matching, params.sources,
                           params.fast_matching, params.no_strict,
-                          params.interactive)
+                          params.interactive, params.log_path)
 
         return code
 
     def unify(self, matching=None, sources=None,
               fast_matching=False, no_strict_matching=False,
-              interactive=False):
+              interactive=False, log_path=None):
```

Comment:
`log_path` is not a clear name for this parameter. For its name, looks like it's doing something different.